src/annotations/standard/PropertyAnnotation.php

```diff
@@ -14,12 +14,98 @@
 namespace mindplay\annotations\standard;
 
 use mindplay\annotations\Annotation;
+use mindplay\annotations\IAnnotationParser;
+use mindplay\annotations\AnnotationException;
+use mindplay\annotations\IAnnotationFileAware;
+use mindplay\annotations\AnnotationFile;
 
 /**
  * Defines a magic/virtual property and it's type
  *
- * @todo implement this
+ * @usage('class'=>true, 'inherited'=>true)
  */
-class PropertyAnnotation extends Annotation
+class PropertyAnnotation extends Annotation implements IAnnotationParser, IAnnotationFileAware
 {
+    /**
+     * Specifies the property type
+     *
+     * @var string 
+     */
+    public $type;
+    
+    /**
+     * Specifies the property name
+     *
+     * @var string
+     */
+    public $name;
+    
+    /**
+     * Specifies the property description
+     *
+     * @var string 
+     */
+    public $description;
+
+    /**
+     * Annotation file.
+     *
+     * @var AnnotationFile
+     */
+    protected $file;
+
+    /**
+     * @param string $value The raw string value of the Annotation.
+     *
+     * @return array An array of Annotation properties.
+     */
+    public static function parseAnnotation($value)
+    {
+        $parts = explode(' ', trim($value), 3);
+
+        if (\sizeof($parts) < 2) {
```

Comment:
I doubt that `\function_name()` syntax (namespaced function calls) would work in all supported PHP versions. Since build haven't failed then obviously annotation engine itself is tested, but not each of annotation classes individually.